engine_wrapper.py

```diff
@@ -37,7 +37,7 @@ def create_engine(config):
         raise ValueError(
             f"    Invalid engine type: {engine_type}. Expected xboard, uci, or homemade.")
     options = remove_managed_options(cfg.lookup(f"{engine_type}_options") or {})
-    logger.debug(f"Starting engine: {' '.join(commands)}")
+    logger.debug(f"Starting engine: '{commands}'")
```

Comment:
Delete the single quotes in this string to get correct coloring in the console log.